syscalls.go

```diff
@@ -88,7 +88,10 @@ var haveMapMutabilityModifiers = internal.NewFeatureTest("read- and write-only m
 		MapFlags:   unix.BPF_F_RDONLY_PROG,
 	})
 	if err != nil {
-		return internal.ErrNotSupported
+		if errors.Is(err, unix.EINVAL) {
```

Comment:
Same comment as elsewhere, is EINVAL more meaningful here?

Reply:
I believe we could just return the `err` too if `err != nil`

Reply:
As per above, let's not change this.